src/styles/theme.js

```diff
@@ -0,0 +1,95 @@
+import { css } from "styled-components";
+
+const breakpoints = {
+  tablet: 1040,
+  mobile: 640
+};
+
+const media = Object.keys(breakpoints).reduce((acc, label) => {
+  acc[label] = (...args) => css`
+    @media screen and (max-width: ${breakpoints[label] / 16}em) {
+      ${css(...args)}
+    }
+  `;
+
+  return acc;
+}, {});
+
+const theme = {
+  color: {
+    accent: {
+      primary: "#226ce7",
+      secondary: "#f42e4f"
+    },
+    background: {
+      base: "#fff",
+      light: "#f5f5f5"
+    },
+    text: {
+      primary: "#222",
+      base: "#666",
+      light: "#aaa",
+      lighter: "#fff",
+      money: "#3ccb3c"
+    },
+    elements: {
+      icon: "#ddd",
+      border: "#ddd",
+      line: "#eee",
+      menu: "#666",
+      dropdown: "#222",
+      tickmark: "#3ccb3c",
+    },
+  },
+  font: {
+    family: {
+      base: "\"Lato\", sans-serif"
+    },
+    size: {
+      base: "14px",
```

Comment:
could we use 10px for easier rem usage? or 16px because <16px is illegible

Reply:
We could go with 16px as base. But personally I disagree, Github base font size is 14px for example. ;)

Reply:
go with 16px, check Google Lighthouse requirement